users/models.py

```diff
@@ -0,0 +1,13 @@
+from django.db                   import models
+from dr_martens.time_stamp_model import TimeStampModel
```

Comment:
```suggestion
from django.db                   import models

from dr_martens.time_stamp_model import TimeStampModel
```
외부 모듈과 사용자 모듈 사이에 공백 추가해 주세요.

Reply:
리뷰 감사합니다.
피드백 반영하여 코드 수정하였습니다.